### PR TITLE
Review scaffolding

### PR DESCRIPTION
**Pull request by alexcodecai**

### CODE REVIEW

**Comment by eliu390** [comment]
Hey Alex, create a file called ".gitignore" in the root directory of the project, put "node_modules/" in the file, and save and push it. That way the repo won't have thousands of files from npm. Otherwise everything looks well done.

**Comment by alexcodecai** [comment]
addressed comments

File: opentable_rv/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+client/dist/bundle.js
+node_modules/
```
Comment (velezjose) [inline opentable_rv/.gitignore]: Please add a newline

File: opentable_rv/client/src/ReviewEntry.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      <div className="txn-data">{props.list.username}</div>
+      <div className="txn-data">{props.list.place}</div>
+      <div className="txn-data">{props.list.DinedDate}</div>
+      <div className="txn-data">{props.list.postedReview}</div>
+    </div>
+  )
+
+}
+
+export default ReviewEntry;
```
Comment (velezjose) [inline opentable_rv/client/src/ReviewEntry.jsx]: Newline here too

File: opentable_rv/client/src/index.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+//import App from './App.jsx';
+import $ from 'jquery';
+import { ajax } from 'jquery';
+import ReviewList from './ReviewList.jsx';
+class App extends React.Component {
```
Comment (velezjose) [inline opentable_rv/client/src/index.js]: I would add a space between the import statements and the class declaration

File: opentable_rv/client/src/index.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        this.getReviews = this.getReviews.bind(this)
+    }
+
+    componentDidMount() {
+        this.getReviews();
+    }
+
+
+    getReviews() {
+        $.get('/api/reviewlist', (rows) => {
```
Comment (velezjose) [inline opentable_rv/client/src/index.js]: This is good. Fyi you could also explore using `fetch` or `axios`.

File: opentable_rv/db/config.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
```
Comment (velezjose) [inline opentable_rv/db/config.js]: You should check this, but I would have the `config.js` file in your `.gitignore`

File: opentable_rv/db/index.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
+const mysql = require('mysql');
+const mysqlConfig = require('./config.js');
+const faker = require('faker');
+const connection = mysql.createConnection(mysqlConfig);
+
+const randomDate = function(start, end) {
```
Comment (velezjose) [inline opentable_rv/db/index.js]: I would suggest if you used ES6 syntax for functions! Like in this case, 
```
const randomDate =(start, end) => {
 ...
};
```

File: opentable_rv/webpack.config.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-react', '@babel/preset-env']
+            
+          }
+        },
+      },
+    ],
+  },
+};
```
Comment (velezjose) [inline opentable_rv/webpack.config.js]: newline

**Pull request merged**